src/mlpack/methods/reinforcement_learning/q_networks/simple_dqn.hpp

```diff
@@ -45,13 +48,18 @@ class SimpleDQN
    * @param h2 Number of neurons in hiddenlayer-2.
    * @param outputDim Number of neurons in output layer.
    * @param isNoisy Specifies whether the network needs to be of type noisy.
+   * @param init Specifies the initilization rule for the network.
+   * @param outputLayer Specifies the output layer type for network.
    */
   SimpleDQN(const int inputDim,
             const int h1,
             const int h2,
             const int outputDim,
-            const bool isNoisy = false):
-      network(MeanSquaredError<>(), GaussianInitialization(0, 0.001)),
+            const bool isNoisy = false,
+            InitType init = GaussianInitialization(0, 0.001),
+            OutputLayerType outputLayer = OutputLayerType()
```

Comment:
Hey there, I don't think this is needed. Take a look at #2476. Thanks.

Reply:
In #2476, how can one set the initialization hyperparameters? I think `GaussianInitialization` uses deault values in that case.